code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm

```diff
@@ -301,6 +301,7 @@
 	var/heal_cost = 100
 	var/heal_amount = 150
 
+	//Aren't these shield values obsolete since Warden had her shield ability removed?
```

Comment:
Remove them if they are

Reply:
I'm a bit afraid to; I don't want to completely remove portions of code unless I get the approval of a maintainer.

Edit: I tried, but then something broke so I am leaving it alone.

Reply:
If it causes errors then it's needed

Reply:
As Zonespace says. Either remove them or remove that line, there's no real point to just add a "maybe" comment

Reply:
Fixed.